README.md

```diff
@@ -49,17 +46,20 @@ For instructions see the following links:
 | [Matlab](https://www.mathworks.com/products/matlab.html) | >=2015b      |
 | or [octave](https://www.gnu.org/software/octave/)        | 4.?          |
 
-The exact version required for this to work but it is known to work with:
+Tested:
 -   matlab 2015b or octave 4.2.2 and PTB 3.0.14.
 
 ##  2. <a name='Codeguidestyle'></a>Code guidestyle
 
 We use the `camelCase` to more easily differentiates our functions from the ones from PTB that use a `PascalCase`.
-We use the following regular expression for function names: `[a-z]+(([A-Z]|[0-9]){1}[a-z]+)*`.
 
-We keep the McCabe complexity as reported by the [check_my_code function](https://github.com/Remi-Gau/check_my_code) below 15.
+In practice, we use the following regular expression for function names: `[a-z]+(([A-Z]|[0-9]){1}[a-z]+)*`.
 
-We use the [MISS_HIT linter](https://florianschanda.github.io/miss_hit/style_checker.html) to automatically fix some linting issues. The code style and quality is also checked during the continuous integration.
+We keep the McCabe complexity as reported by the [check_my_code function](https://github.com/Remi-Gau/check_my_code) below 15 or the [MISS_HIT code checker](https://florianschanda.github.io/miss_hit)
+
+We use the [MISS_HIT linter](https://florianschanda.github.io/miss_hit/style_checker.html) to automatically fix some linting issues. 
+
+The code style and quality is also checked during the [continuous integration](./.travis.yml).
 
 ##  3. <a name='Howtoinstall'></a>How to install
 
```

Comment:
>     1. I don't think that the `matlab` command would work on any computer
> 
>     2. it should be `addpath(genpath(fullfile(pwd)))` to add all the sub folders, isn't it?

```bash
matlab -nojvm -nosplash -r "addpath(genpath(fullfile(pwd, 'src'))); savepath(); path(); exit();"
```

I have just typed this in the terminal when inside my CPP_PTB folder and it works fine: could you check it?

Reply:
> line 101 same problems plus, maybe, better to not add it in the matlab path. In case I have two experiments using different versions of this repo, would I encounter problems of func duplicates?

I actually agree: will move this whole section at the very end with strong message of disapproval for doing this

Reply:
oh of course.
I think I must have set up an alias actually

it would work by default with octave thouhg... Just saying...